config/initializers/job_configurations.rb

```diff
@@ -6,9 +6,11 @@
   name: 'Send GPO letter',
   interval: 24 * 60 * 60,
   timeout: 300,
-  callback: lambda {
+  callback: lambda do
+    UspsDailyTestSender.new.run
```

Comment:
It seemed easier to add this step to the existing daily job, rather than worry about race conditions across multiple daily jobs? I did add the top-level rescue to minimize the chances that this job will break the actual upload run